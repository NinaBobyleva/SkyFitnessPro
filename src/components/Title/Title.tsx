export function Title({title}: {title: string}) {
  return (
    <h1 className="font-roboto-500 text-2xl lg:text-6xl font-medium text-black mb-[10px] lg:mb-6">
      {title}
    </h1>
  );
}
