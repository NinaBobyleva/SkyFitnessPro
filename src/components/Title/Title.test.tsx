import renderer from "react-test-renderer";
import { Title } from "./Title";

describe("Компонент Title", () => {

  it("Успешный рендер Title", () => {
    const component = renderer.create(<Title title="Hello" />).toJSON();
    expect(component).toMatchSnapshot();
  });
});