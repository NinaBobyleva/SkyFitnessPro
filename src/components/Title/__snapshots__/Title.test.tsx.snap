// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент Title Успешный рендер Title 1`] = `
<h1
  className="font-roboto-500 text-2xl lg:text-6xl font-medium text-black mb-[10px] lg:mb-6"
>
  Hello
</h1>
`;
